### PR TITLE
Added social round buttons

### PR DESCRIPTION
**Pull request by Neeraj3508**
Issue:

### CODE REVIEW

**Comment by Neeraj3508** [comment]
@shahednasser you can now check it is showing no conflicts

**Review by shahednasser — CHANGES_REQUESTED** [review]
It all looks great, there's just one problem with the font. Previously, there was a rule in `assets/css/index.css` something like:
```
.social-btn {
font-family: 'Syne', sans-serif;
}
```

I'm not sure why it's gone now but please add it again.

**Comment by Neeraj3508** [comment]
@shahednasser you can check all the changes made + the one requested

File: assets/css/index.css [hunk 1]
```diff
@@ -306,8 +306,8 @@ hr {
 }
 
 .social-btn {
-  font-family: Syne, sans-serif;
-}
+  font-family: 'Syne', sans-serif;
```
Comment (shahednasser) [inline assets/css/index.css:309]: Please add "!important" to it as it is not applied.

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by Neeraj3508** [comment]
@shahednasser done, added !important to index.html

**Comment by shahednasser** [comment]
Great work thank you!

**Comment by Neeraj3508** [comment]
😊

**Pull request merged**